test/node/parser/calculate_size.test.ts

```diff
@@ -24,4 +24,16 @@ describe('calculateSize()', () => {
       })
     ).to.throw(BSONVersionError, /Unsupported BSON version/i);
   });
+
+  it('returns 8 bytes (+7 meta bytes) size for a bigint value', function () {
```

Comment:
```suggestion
  it('returns 8 bytes (+8 meta bytes) size for a bigint value', function () {
```

right? 8 for the bigint, and then 1 for the 'long' type tag, 1+1 for the name, 4 for the document length, 1 trailing nul byte to finish the document?